src/rust/bitbox02-sys/Cargo.toml

```diff
@@ -23,3 +23,6 @@ license = "Apache-2.0"
 
 [dependencies]
 util = {path = "../util"}
+
+[build-dependencies]
+cc = {version="1.2.31", features=["parallel"]}
```

Comment:
We already have cc=1.0.79 in the vendor/ folder, this adds another copy. Maybe we can avoid it and only use one version?